tests/Makefile.am

```diff
@@ -111,10 +111,12 @@ TESTS = \
 	test-of-option.sh \
 	test-on-option.sh \
 	test-print-source.sh \
+	test-phony-win2k22.sh \
 	test-rhbz1232192.sh \
```

Comment:
I believe this ends up adding `test-phony-win2k22.sh` twice to TESTS.

Reply:
oops yes. all fixed now